### PR TITLE
rp-pppoe: improve init scripts

### PR DESCRIPTION
**Pull request by alinnastac**
Input interface parameters using their netifd logical names, fix
interface triggers and add enable boolean parameter in
pppoe_{server,relay} UCI sections.

Signed-off-by: Alin Nastac <alin.nastac@gmail.com>

Maintainer: Daniel Dickinson <lede@cshore.thecshore.com>
Run tested: arm_cortex-a9 openwrt/chaos_calmer

### CODE REVIEW

File: net/rp-pppoe/files/pppoe-relay.init [hunk 1]
```diff
@@ -2,28 +2,27 @@
...
 
 START=50
+STOP=50
 USE_PROCD=1
 
 pppoe_triggers() {
     local cfg="$1"
-    local interface server_interfaces client_interfaces both_interfaces
+    local enabled interface server_interfaces client_interfaces both_interfaces
+    config_get_bool enabled "$cfg" enable 1
```
Comment (dedeckeh) [inline net/rp-pppoe/files/pppoe-relay.init]: Force consistency with other implementations call the parameter enabled

File: net/rp-pppoe/files/pppoe-relay.init [hunk 2]
```diff
@@ -2,28 +2,27 @@
...
+    for interface in $server_interfaces $client_interfaces $both_interfaces; do
+	procd_add_reload_interface_trigger $interface
     done
 }
 
 pppoe_relay_instance() {
     local cfg="$1"
-    local interface server_interfaces client_interfaces both_interfaces maxsessions timeout OPTIONS
+    local enabled interface device server_interfaces client_interfaces both_interfaces maxsessions timeout OPTIONS
+    config_get_bool enabled "$cfg" enable 1
```
Comment (dedeckeh) [inline net/rp-pppoe/files/pppoe-relay.init]: Force consistency with other implementations call the parameter enabled

**Comment by dedeckeh** [comment]
Could you split this up in different commits (support for logical interface names, interface trigger fix, support for enabled uci parameters, indenting changes) ? This will make the review easier

**Comment by cshoredaniel** [comment]
@dedeckeh 
Er sorry I meant to send a remove myself as maintainer sometime ago; I had personal problems that complicated my relationship with OpenWrt/LEDE (you may have noticed at the time); I'm not sure if I'll be back in the future, but at the moment I'm dealing with a lot of offline stuff.

**Pull request merged**